cleanup-passes/travis.ts

```diff
@@ -32,56 +31,109 @@ async function cleanupTravisConfig(element: ElementRepo): Promise<void> {
 
   let travis = yaml.safeLoad(travisConfigBlob);
 
-  let needsReview = false;
+  const tools: string[] = [
+    'bower',
+    'polylint',
+    'web-component-tester'
+  ];
+
+  const beforeScript: string[] = [
+    `npm install -g ${tools.join(" ")}`,
+    'bower install',
+    'polylint'
+  ];
 
   // update travis config
   // Add polylint to all elements
   if (Array.isArray(travis.before_script)) {
-    const installStep = 'npm install polylint';
-    const runStep = 'polylint';
+    const bs: string[] = travis.before_script;
+    if (!bs.reduce(
```

Comment:
since `travis.before_script` is type `any`, typescript freaks out when I call reduce on it. It looks ok with explicit types for the arguments in reduce though

`(acc: boolean, s: string, idx: number) => ...`